docs/index.markdown

```diff
@@ -31,7 +31,8 @@ Join us for our **weekly and monthly events** or just stop by to say hello!
 
 - **Heroclix:** Tuesday @ 6:30 pm and Saturday @ 3:00 pm
 - **Star Wars: Legion and Shatterpoint** Tuesday @ 6:30 pm
--**Magic the Gathering: Foundations Draft:** Friday @ 6:30 PM
+- **Magic the Gathering: Standard Showdown:** Friday @ 6:00 pm
+-**Magic the Gathering: Foundations Draft:** Friday @ 6:30 pm
```

Comment:
I'll fix this, but there should be a space here to keep it consistent with the others.